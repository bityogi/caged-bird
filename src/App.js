import React, { Component } from 'react';
import { withRouter } from 'react-router-dom';
import { connect } from 'react-redux';
import compose from 'recompose/compose';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import autoprefixer from 'material-ui/utils/autoprefixer';
import CircularProgress from 'material-ui/CircularProgress';
import withWidth from 'material-ui/utils/withWidth';

import { Header, Notification, Alert } from 'components';
import Routes from './routes';
import theme from './theme';


import './App.css';

const styles = {
  wrapper: {
        // Avoid IE bug with Flexbox, see #467
        display: 'flex',
        flexDirection: 'column',
    },
    main: {
        display: 'flex',
        flexDirection: 'column',
        minHeight: '100vh',
    },
    body: {
        backgroundColor: '#edecec',
        display: 'flex',
        flex: 1,
        overflowY: 'hidden',
    },
    content: {
        flex: 1,
        paddingTop: '1em',
        paddingLeft: '1em',
        paddingRight: '1em',
    },
    loader: {
        position: 'absolute',
        top: 0,
        right: 0,
        margin: 16,
        zIndex: 1200,
    },
};

const prefixedStyles = {};

class App extends Component {

  render() {
    const {
      width,
      isLoading
    } = this.props;

    const muiTheme = getMuiTheme(theme);
    if (!prefixedStyles.main) {
      const prefix = autoprefixer(muiTheme);
      prefixedStyles.wrapper = prefix(styles.wrapper);
      prefixedStyles.main = prefix(styles.main);
      prefixedStyles.body = prefix(styles.body);
      prefixedStyles.bodySmall = prefix(styles.bodySmall);
      prefixedStyles.content = prefix(styles.content);
      prefixedStyles.contentSmall = prefix(styles.contentSmall);
    }

    return (
      <MuiThemeProvider muiTheme={muiTheme}>
        <div style={prefixedStyles.wrapper}>
          <div style={prefixedStyles.main}>
            {width !== 1 && <Header />}

            <div
             className="body"
             style={
               width === 1 ? ( prefixedStyles.bodySmall) : (prefixedStyles.body)
             }
            >

              <div style={
                    width === 1 ? (prefixedStyles.contentSmall) : (prefixedStyles.content)
                  }
                >
                <Routes />


              </div>


            </div>

            <Alert />
            <Notification />
            {isLoading && (
               <CircularProgress
                   className="app-loader"
                   color="#fff"
                   size={width === 1 ? 20 : 30}
                   thickness={2}
                   style={styles.loader}
               />
           )}
          </div>
        </div>
      </MuiThemeProvider>
    );
  }
}

const mapStateToProps = (state) => ({
  users : state.users,
  isLoading: state.admin.loading > 0,
})

const enhance = compose(
  connect(mapStateToProps, null),
  withWidth()
);

export default withRouter(enhance(App));
