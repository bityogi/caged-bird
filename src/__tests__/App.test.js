import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router';

import App from 'App';
import store from 'store';

describe('App', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <Provider store={store}>
        <MemoryRouter initialEntries={['/', '/landing']}>
          <App />
        </MemoryRouter>

      </Provider>

    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });

});
