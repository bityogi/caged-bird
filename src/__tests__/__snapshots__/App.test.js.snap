// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
        "minHeight": "100vh",
      }
    }
  >
    <div>
      <div
        className={undefined}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0,0,0,0)",
            "backgroundColor": "#00838f",
            "boxSizing": "border-box",
            "display": "flex",
            "height": 56,
            "justifyContent": "space-between",
            "muiPrepared": true,
            "padding": "0px 24px",
          }
        }
      >
        <div
          className={undefined}
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "justifyContent": "space-between",
              "marginLeft": -24,
              "marginRight": undefined,
              "muiPrepared": true,
              "position": "relative",
            }
          }
        >
          <img
            alt="Caged-Bird"
            height={50}
            onClick={[Function]}
            src="Cagebird.png"
            style={
              Object {
                "borderRadius": 5,
                "marginBottom": 3,
                "marginLeft": 5,
                "marginRight": 5,
                "marginTop": 3,
              }
            }
            width={60}
          />
          <img
            alt="Caged-Bird"
            height={56}
            onClick={[Function]}
            src="Artboard 3 copy 11.png"
            style={
              Object {
                "marginBottom": 3,
                "marginRight": 50,
                "marginTop": 3,
              }
            }
            width={200}
          />
        </div>
        <div
          className={undefined}
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "justifyContent": "space-between",
              "marginLeft": undefined,
              "marginRight": undefined,
              "muiPrepared": true,
              "position": "relative",
            }
          }
        >
          <img
            alt="Caged-Bird"
            height={48}
            src="LOGO_WHITE.png"
            style={
              Object {
                "borderRadius": 5,
                "marginBottom": 3,
                "marginLeft": 5,
                "marginRight": 5,
                "marginTop": 3,
              }
            }
            width={120}
          />
        </div>
      </div>
    </div>
    <div
      className="body"
      style={
        Object {
          "backgroundColor": "#edecec",
          "display": "flex",
          "flex": 1,
          "overflowY": "hidden",
        }
      }
    >
      <div
        style={
          Object {
            "flex": 1,
            "paddingLeft": "1em",
            "paddingRight": "1em",
            "paddingTop": "1em",
          }
        }
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#00838f",
              "display": "flex",
              "flexDirection": "column",
              "justifyContent": "center",
              "minHeight": "100vh",
            }
          }
        >
          <div
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": 2,
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "minWidth": 300,
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                style={
                  Object {
                    "margin": "1em",
                    "textAlign": "center ",
                  }
                }
              >
                <div
                  className={undefined}
                  size={60}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#880e4f",
                      "borderRadius": "50%",
                      "color": "#ffffff",
                      "display": "inline-flex",
                      "fontSize": 30,
                      "height": 60,
                      "justifyContent": "center",
                      "muiPrepared": true,
                      "userSelect": "none",
                      "width": 60,
                    }
                  }
                >
                  <svg
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={
                      Object {
                        "color": "#ffffff",
                        "display": "inline-block",
                        "fill": "#ffffff",
                        "fontSize": 36,
                        "height": 36,
                        "margin": 12,
                        "muiPrepared": true,
                        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                        "userSelect": "none",
                        "width": 36,
                      }
                    }
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 17c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm6-9h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM8.9 6c0-1.71 1.39-3.1 3.1-3.1s3.1 1.39 3.1 3.1v2H8.9V6zM18 20H6V10h12v10z"
                    />
                  </svg>
                </div>
              </div>
              <form
                onSubmit={[Function]}
              >
                <div
                  style={
                    Object {
                      "padding": "0 1em 1em 1em",
                    }
                  }
                >
                  <div
                    style={
                      Object {
                        "display": "flex",
                      }
                    }
                  >
                    <div
                      style={
                        Object {
                          "borderColor": "#00838f",
                          "borderRadius": "5px",
                          "borderStyle": "solid",
                          "borderWidth": "1px 1px 3px 1px",
                          "flex": 1,
                          "margin": 10,
                          "padding": 10,
                        }
                      }
                    >
                      <div>
                        <label>
                          Operator 1
                        </label>
                      </div>
                      <div>
                        <div
                          className={undefined}
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "cursor": "auto",
                              "display": "inline-block",
                              "fontFamily": "Roboto, sans-serif",
                              "fontSize": 16,
                              "height": 72,
                              "lineHeight": "24px",
                              "muiPrepared": true,
                              "position": "relative",
                              "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                              "width": 256,
                            }
                          }
                        >
                          <label
                            className={undefined}
                            htmlFor="username1"
                            onClick={undefined}
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.3)",
                                "lineHeight": "22px",
                                "muiPrepared": true,
                                "pointerEvents": "none",
                                "position": "absolute",
                                "top": 38,
                                "transform": "scale(1) translate(0, 0)",
                                "transformOrigin": "left top",
                                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                "userSelect": "none",
                                "zIndex": 1,
                              }
                            }
                          >
                            Operator 1
                          </label>
                          <input
                            disabled={false}
                            id="username1"
                            name="username1"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onDragStart={[Function]}
                            onDrop={[Function]}
                            onFocus={[Function]}
                            style={
                              Object {
                                "WebkitOpacity": 1,
                                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                                "appearance": "textfield",
                                "backgroundColor": "rgba(0,0,0,0)",
                                "border": "none",
                                "boxSizing": "border-box",
                                "color": "rgba(0, 0, 0, 0.87)",
                                "cursor": "inherit",
                                "font": "inherit",
                                "height": "100%",
                                "marginTop": 14,
                                "muiPrepared": true,
                                "outline": "none",
                                "padding": 0,
                                "position": "relative",
                                "width": "100%",
                              }
                            }
                            type="text"
                            value=""
                          />
                          <div>
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 1,
                                  "borderColor": "#e0e0e0",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "width": "100%",
                                }
                              }
                            />
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 2,
                                  "borderColor": "#00838f",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "transform": "scaleX(0)",
                                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                  "width": "100%",
                                }
                              }
                            />
                          </div>
                        </div>
                      </div>
                      <div>
                        <div
                          className={undefined}
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "cursor": "auto",
                              "display": "inline-block",
                              "fontFamily": "Roboto, sans-serif",
                              "fontSize": 16,
                              "height": 72,
                              "lineHeight": "24px",
                              "muiPrepared": true,
                              "position": "relative",
                              "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                              "width": 256,
                            }
                          }
                        >
                          <label
                            className={undefined}
                            htmlFor="password1"
                            onClick={undefined}
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.3)",
                                "lineHeight": "22px",
                                "muiPrepared": true,
                                "pointerEvents": "none",
                                "position": "absolute",
                                "top": 38,
                                "transform": "scale(1) translate(0, 0)",
                                "transformOrigin": "left top",
                                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                "userSelect": "none",
                                "zIndex": 1,
                              }
                            }
                          >
                            Password
                          </label>
                          <input
                            disabled={false}
                            id="password1"
                            name="password1"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onDragStart={[Function]}
                            onDrop={[Function]}
                            onFocus={[Function]}
                            style={
                              Object {
                                "WebkitOpacity": 1,
                                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                                "appearance": "textfield",
                                "backgroundColor": "rgba(0,0,0,0)",
                                "border": "none",
                                "boxSizing": "border-box",
                                "color": "rgba(0, 0, 0, 0.87)",
                                "cursor": "inherit",
                                "font": "inherit",
                                "height": "100%",
                                "marginTop": 14,
                                "muiPrepared": true,
                                "outline": "none",
                                "padding": 0,
                                "position": "relative",
                                "width": "100%",
                              }
                            }
                            type="password"
                            value=""
                          />
                          <div>
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 1,
                                  "borderColor": "#e0e0e0",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "width": "100%",
                                }
                              }
                            />
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 2,
                                  "borderColor": "#00838f",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "transform": "scaleX(0)",
                                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                  "width": "100%",
                                }
                              }
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      style={
                        Object {
                          "borderColor": "#00838f",
                          "borderRadius": "5px",
                          "borderStyle": "solid",
                          "borderWidth": "1px 1px 3px 1px",
                          "flex": 1,
                          "margin": 10,
                          "padding": 10,
                        }
                      }
                    >
                      <div>
                        <label>
                          Operator 2
                        </label>
                      </div>
                      <div>
                        <div
                          className={undefined}
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "cursor": "auto",
                              "display": "inline-block",
                              "fontFamily": "Roboto, sans-serif",
                              "fontSize": 16,
                              "height": 72,
                              "lineHeight": "24px",
                              "muiPrepared": true,
                              "position": "relative",
                              "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                              "width": 256,
                            }
                          }
                        >
                          <label
                            className={undefined}
                            htmlFor="username2"
                            onClick={undefined}
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.3)",
                                "lineHeight": "22px",
                                "muiPrepared": true,
                                "pointerEvents": "none",
                                "position": "absolute",
                                "top": 38,
                                "transform": "scale(1) translate(0, 0)",
                                "transformOrigin": "left top",
                                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                "userSelect": "none",
                                "zIndex": 1,
                              }
                            }
                          >
                            Operator 2
                          </label>
                          <input
                            disabled={false}
                            id="username2"
                            name="username2"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onDragStart={[Function]}
                            onDrop={[Function]}
                            onFocus={[Function]}
                            style={
                              Object {
                                "WebkitOpacity": 1,
                                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                                "appearance": "textfield",
                                "backgroundColor": "rgba(0,0,0,0)",
                                "border": "none",
                                "boxSizing": "border-box",
                                "color": "rgba(0, 0, 0, 0.87)",
                                "cursor": "inherit",
                                "font": "inherit",
                                "height": "100%",
                                "marginTop": 14,
                                "muiPrepared": true,
                                "outline": "none",
                                "padding": 0,
                                "position": "relative",
                                "width": "100%",
                              }
                            }
                            type="text"
                            value=""
                          />
                          <div>
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 1,
                                  "borderColor": "#e0e0e0",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "width": "100%",
                                }
                              }
                            />
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 2,
                                  "borderColor": "#00838f",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "transform": "scaleX(0)",
                                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                  "width": "100%",
                                }
                              }
                            />
                          </div>
                        </div>
                      </div>
                      <div>
                        <div
                          className={undefined}
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "cursor": "auto",
                              "display": "inline-block",
                              "fontFamily": "Roboto, sans-serif",
                              "fontSize": 16,
                              "height": 72,
                              "lineHeight": "24px",
                              "muiPrepared": true,
                              "position": "relative",
                              "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                              "width": 256,
                            }
                          }
                        >
                          <label
                            className={undefined}
                            htmlFor="password2"
                            onClick={undefined}
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.3)",
                                "lineHeight": "22px",
                                "muiPrepared": true,
                                "pointerEvents": "none",
                                "position": "absolute",
                                "top": 38,
                                "transform": "scale(1) translate(0, 0)",
                                "transformOrigin": "left top",
                                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                "userSelect": "none",
                                "zIndex": 1,
                              }
                            }
                          >
                            Password
                          </label>
                          <input
                            disabled={false}
                            id="password2"
                            name="password2"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onDragStart={[Function]}
                            onDrop={[Function]}
                            onFocus={[Function]}
                            style={
                              Object {
                                "WebkitOpacity": 1,
                                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                                "appearance": "textfield",
                                "backgroundColor": "rgba(0,0,0,0)",
                                "border": "none",
                                "boxSizing": "border-box",
                                "color": "rgba(0, 0, 0, 0.87)",
                                "cursor": "inherit",
                                "font": "inherit",
                                "height": "100%",
                                "marginTop": 14,
                                "muiPrepared": true,
                                "outline": "none",
                                "padding": 0,
                                "position": "relative",
                                "width": "100%",
                              }
                            }
                            type="password"
                            value=""
                          />
                          <div>
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 1,
                                  "borderColor": "#e0e0e0",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "width": "100%",
                                }
                              }
                            />
                            <hr
                              aria-hidden="true"
                              style={
                                Object {
                                  "borderBottomStyle": "solid",
                                  "borderBottomWidth": 2,
                                  "borderColor": "#00838f",
                                  "borderLeft": "none",
                                  "borderRight": "none",
                                  "borderTop": "none",
                                  "bottom": 8,
                                  "boxSizing": "content-box",
                                  "margin": 0,
                                  "muiPrepared": true,
                                  "position": "absolute",
                                  "transform": "scaleX(0)",
                                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                                  "width": "100%",
                                }
                              }
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    style={
                      Object {
                        "muiPrepared": true,
                        "padding": 8,
                        "position": "relative",
                      }
                    }
                  >
                    <div
                      className={undefined}
                      style={
                        Object {
                          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                          "backgroundColor": "#ffffff",
                          "borderRadius": 2,
                          "boxShadow": undefined,
                          "boxSizing": "border-box",
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fontFamily": "Roboto, sans-serif",
                          "marginRight": 8,
                          "minWidth": "100%",
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                        }
                      }
                    >
                      <button
                        disabled={true}
                        onBlur={[Function]}
                        onClick={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        style={
                          Object {
                            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                            "backgroundColor": "rgb(229, 229, 229)",
                            "border": 10,
                            "borderRadius": 2,
                            "boxSizing": "border-box",
                            "cursor": "default",
                            "display": "inline-block",
                            "fontFamily": "Roboto, sans-serif",
                            "fontSize": "inherit",
                            "fontWeight": "inherit",
                            "height": 36,
                            "lineHeight": "36px",
                            "margin": 0,
                            "muiPrepared": true,
                            "outline": "none",
                            "padding": 0,
                            "position": "relative",
                            "textAlign": "center",
                            "textDecoration": "none",
                            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                            "verticalAlign": null,
                            "width": "100%",
                          }
                        }
                        tabIndex={-1}
                        type="submit"
                      >
                        <div
                          style={
                            Object {
                              "backgroundColor": false,
                              "borderRadius": 2,
                              "height": 36,
                              "muiPrepared": true,
                              "top": 0,
                              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                            }
                          }
                        >
                          <span
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.3)",
                                "fontSize": 14,
                                "fontWeight": 500,
                                "letterSpacing": 0,
                                "margin": 0,
                                "muiPrepared": true,
                                "opacity": 1,
                                "paddingLeft": 16,
                                "paddingRight": 16,
                                "position": "relative",
                                "textTransform": "uppercase",
                                "userSelect": "none",
                              }
                            }
                          >
                            Sign In
                          </span>
                        </div>
                      </button>
                    </div>
                  </div>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      open={false}
      style={
        Object {
          "bottom": 0,
          "display": "flex",
          "left": "50%",
          "muiPrepared": true,
          "position": "fixed",
          "transform": "translate(-50%, 48px)",
          "transition": "transform 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms, visibility 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "visibility": "hidden",
          "zIndex": 2900,
        }
      }
    >
      <div
        style={
          Object {
            "backgroundColor": "rgba(0, 0, 0, 0.87)",
            "borderRadius": 2,
            "flexGrow": 0,
            "fontFamily": "Roboto, sans-serif",
            "height": "auto",
            "lineHeight": "20px",
            "maxWidth": 568,
            "minWidth": 288,
            "muiPrepared": true,
            "padding": 14,
            "whiteSpace": "pre-line",
            "width": "auto",
          }
        }
        width={3}
      >
        <div
          style={
            Object {
              "color": "#ffffff",
              "fontSize": 14,
              "muiPrepared": true,
              "opacity": 0,
              "transition": "opacity 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            }
          }
        >
          <span />
        </div>
      </div>
    </div>
  </div>
</div>
`;
